src/components/pages/groups-page/groups-page.tsx

```diff
@@ -39,7 +42,14 @@ export const GroupsPage = () => {
     requestAccess(org).then();
   }
 
+  function updateAccessRequestsLoading(org: ApiOrg, isLoading: boolean) {
+    const accessRequestsLoadingClone = { ...accessRequestsLoading };
+    accessRequestsLoadingClone[org.id] = isLoading;
+    setAccessRequestsLoading(accessRequestsLoadingClone);
+  }
```

Comment:
If I'm understanding React right, I think I should make a clone before I update the state on an object like I did here. With that said, not completely sure if this is necessary or not.